cpp/tensorrt_llm/kernels/layernormKernels.cu

```diff
@@ -57,16 +58,26 @@ __inline__ __device__ Tf compute_layernorm(Tf val, float s_mean, float s_varianc
  *           amax per row. A final pass scales to int8 accordingly, and writes output to
  *           normed_output_quant.
  */
-template <typename T, bool USE_DIFF_OF_SQUARES = false>
+template <typename T, typename QuantT, bool USE_SHMEM, bool USE_DIFF_OF_SQUARES = false>
```

Comment:
in the function docstring above, need to change `use_shmem` to `USE_SHMEM`. Also need to explain template param `QuantT` -- this is also missing in `rmsnormKernels.cu`, so we'd better add there too